internal/api/organizationconfig.go

```diff
@@ -0,0 +1,90 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/joshsoftware/peerly-backend/internal/app/organizationConfig"
+	"github.com/joshsoftware/peerly-backend/internal/pkg/apperrors"
+	"github.com/joshsoftware/peerly-backend/internal/pkg/dto"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+
+
+func getOrganizationConfigHandler(orgSvc organizationConfig.Service) http.HandlerFunc {
+	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+
+		orgConfig, err := orgSvc.GetOrganizationConfig(req.Context())
+		if err != nil {
+			logger.Errorf("Error while fetching organization: %v",err)
+			dto.ErrorRepsonse(rw, err)
+			return
+		}
+
+		dto.SuccessRepsonse(rw, http.StatusOK, "organization config fetched successfully",orgConfig)
+	})
+}
+
+
+func createOrganizationConfigHandler(orgSvc organizationConfig.Service) http.HandlerFunc {
```

Comment:
If we will be having only 1 organization for ever then this code can be removed so that no one accidentlly create new org.

Reply:
we can comment out this route.
A check is applied to check whether the organization configuration already exists in the create organization configuration